src/index.unit.tsx

```diff
@@ -1,7 +1,6 @@
 import fs from 'fs'
 
 import * as kirk from './index'
-import * as icon from './icon/index'
```

Comment:
Why are you removing single icon stories?
That's the only way we have to use knobs to test modifing their props.

Reply:
It's the test file.
Knobs still up and running